packages/plugin-ext/src/main/node/plugin-mgmt-cli-contribution.ts

```diff
@@ -0,0 +1,64 @@
+// *****************************************************************************
+// Copyright (C) 2023 STMicroelectronics and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { Argv, Arguments } from '@theia/core/shared/yargs';
+import { CliContribution } from '@theia/core/lib/node/cli';
+import { HostedPluginDeployerHandler } from '../../hosted/node/hosted-plugin-deployer-handler';
+import { PluginType } from '../../common';
+
+@injectable()
+export class PluginMgmtCliContribution implements CliContribution {
```

Comment:
Minor: We usually don't abbreviate names, see also [the coding guidelines](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#whole-words-names).